### ISSUE TITLE
(2) Use and organize SCSS in Rails

### ISSUE THREAD

**Issue by Akalinsky** [Akalinsky/tc359_rails_final#2]
@chrisvfritz

I think I have completed all of these requirements. A bit fuzzy on the "professionally styled" but I would like to make it look a bit more aesthetically appealing in class.

**Comment by chrisvfritz** [comment]
I agree that we can improve this. I'd be happy to help you come up with points for improvement in class tomorrow, which will give you an opportunity to write and organize your own styles.

**Comment by Akalinsky** [comment]
updated! :shipit: ?

**Comment by chrisvfritz** [comment]
:+1: :tada: :balloon: :ship: :metal:
